internal/host/host_test.go

```diff
@@ -3280,6 +3280,113 @@ var _ = Describe("Get host by Kube key", func() {
 	})
 })
 
+var _ = Describe("Media disconnection", func() {
+
+	var (
+		ctx          = context.Background()
+		db           *gorm.DB
+		api          API
+		dbName       string
+		config       Config
+		mockEventApi *eventsapi.MockHandler
+		ctrl         *gomock.Controller
+		clusterId    strfmt.UUID
+		hostId       strfmt.UUID
+		infraEnvId   strfmt.UUID
+		host         models.Host
+	)
+
+	BeforeEach(func() {
+		db, dbName = common.PrepareTestDB()
+		config = *defaultConfig
+		ctrl = gomock.NewController(GinkgoT())
+		mockEventApi = eventsapi.NewMockHandler(ctrl)
+		dummy := &leader.DummyElector{}
+		api = NewManager(common.GetTestLog(), db, mockEventApi, nil, nil, nil, nil, &config, dummy, nil, nil)
+		clusterId = strfmt.UUID(uuid.New().String())
+		hostId = strfmt.UUID(uuid.New().String())
+		infraEnvId = strfmt.UUID(uuid.New().String())
+		host = hostutil.GenerateTestHost(hostId, infraEnvId, clusterId, models.HostStatusInsufficient)
+		Expect(db.Create(&host).Error).ShouldNot(HaveOccurred())
+	})
+
+	AfterEach(func() {
+		common.DeleteTestDB(db, dbName)
+		ctrl.Finish()
+	})
+
+	const errorMessage string = "Failed - " + statusInfoMediaDisconnected
+
+	It("Media disconnection occurred before installation", func() {
```

Comment:
do we have a test for media disconnection during installation?

Reply:
yes